packages/common/src/ui/layout/tables/components/HeaderRow/HeaderRow.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, ReactNode } from 'react';
+import { TableCell, TableRow, TableSortLabel } from '@mui/material';
+import { KeyOf, ObjectWithStringKeys } from '../../../../../types/utility';
+import { SortRule } from '../../../../../hooks/useSortBy';
+import { SortDesc } from '../../../../icons';
+
+export const HeaderRow: FC = props => (
+  <TableRow
+    {...props}
+    sx={{
+      display: 'flex',
+      flex: '1 0 auto',
+      height: '60px',
+      paddingLeft: '20px',
+      paddingRight: '20px',
+      alignItems: 'center',
+    }}
+  />
+);
+
+interface HeaderCellProps<T extends ObjectWithStringKeys> {
+  onSortBy?: (sortBy: SortRule<T>) => void;
+  isSortable: boolean;
+  isSorted?: boolean;
+  align?: 'left' | 'right' | 'center';
+  id: KeyOf<T>;
+  direction?: 'asc' | 'desc';
+  children: ReactNode;
+  style: React.CSSProperties;
+}
+
+export const HeaderCell = <T extends ObjectWithStringKeys>({
```

Comment:
:nit I would prefer to have either the `HeaderCell` in its own file, or the file be called `Header` instead of `HeaderRow`

Reply:
I went with `Header`. I actually did `HeaderCell` and `HeaderRow`. And then I went back to `Header`. And I am still unsure.

😢